### PR TITLE
Isu00037 readme.md fails at manage.py runserver

### PR DESCRIPTION
**Pull request by tine3700**
Please review and let me know where you can suggest improvements.

To test and run the system i did this:

```
/>   python manage.py runserver 0.0.0.0:8000

***** WARNING! This is a non-production build *****

***** WARNING! This is a non-production build *****
Watching for file changes with StatReloader
Performing system checks...

System check identified no issues (0 silenced).

You have 33 unapplied migration(s). Your project may not work properly until you apply the migrations for app(s): account, admin, auth, blogs, catalog, contenttypes, qrcode, sessions, sites.
Run 'python manage.py migrate' to apply them.

May 30, 2022 - 13:37:40
Django version 2.2.27, using settings 'config.settings'
Starting development server at http://0.0.0.0:8000/
Quit the server with CONTROL-C.
```

Go to your browser page: 

Example page is:http://localhost:8000/

noisy-atom

### CODE REVIEW

**Comment by nherriot** [comment]
@tine3700 Please resolve conflicts so that I can properly review this pull request. Thank you.

**Comment by nherriot** [comment]
@tine3700 Please properly fill out your pull request description.

Example:
1. What do you do to test out your change. Just run up the system?
2. Please include a cut'n'paste of the terminal of what you get when you run up the system.
3. Did you run up in dev mode, or production mode?
4. If you have changed something in the code which is not obvious please say why you needed to change it.

Thank you.

File: documentation/README.md [hunk 1]
```diff
@@ -149,7 +149,7 @@ localhost:8000
 If you don't have 'postgres' installed on your machine you can install with the following packages:
 ```bash
 /> sudo apt-get install postgresql-contrib
-/> sudo apt-get install postgresql-server-dev-9.3
+/> sudo apt-get install postgresql-server-dev-10
```
Comment (nherriot) [inline documentation/README.md:152]: Why did you upgrade the DB. The issue was not to upgrade the DB, but to upgrade to django 2.2. with python 3.8 ?

File: documentation/README.md [hunk 2]
```diff
@@ -249,7 +249,6 @@ Python 2.7.6 (default, Jun 22 2015, 17:58:13)
 [GCC 4.8.2] on linux2
 Type "help", "copyright", "credits" or "license" for more information.
 (InteractiveConsole)
->>>from django.db import connection
 >>> from django.db import connection
```
Comment (nherriot) [inline documentation/README.md:252]: Good catch. Thank you. :-)

File: noisyatom/catalog/models.py [hunk 3]
```diff
@@ -24,10 +25,13 @@ class Meta:
 
     def __unicode__(self):
         return self.name
-
+'''
     @models.permalink
     def get_absolute_url(self):
         return 'catalog_category', (), {'category_slug': self.slug}
+'''
```
Comment (nherriot) [inline noisyatom/catalog/models.py:32]: Why did you comment out this code?
If it's not needed, should we just remove it? Were you planning on putting it back after some testing?

File: noisyatom/catalog/models.py [hunk 4]
```diff
@@ -64,10 +68,14 @@ class Meta:
 
     def __unicode__(self):
         return self.name
-
+'''
```
Comment (nherriot) [inline noisyatom/catalog/models.py:71]: Same as before. Why commented out?

File: noisyatom/catalog/models.py [hunk 5]
```diff
@@ -104,9 +112,11 @@ class Meta:
...
```
Comment (nherriot) [inline noisyatom/catalog/models.py:115]: Same as before. Why commented out?

File: noisyatom/config/settings.py [hunk 6]
```diff
@@ -41,16 +41,16 @@
     # DATABASES = {
     #     'default': {
     #         'ENGINE': 'django.db.backends.postgresql',
-    #         'NAME': 'na_db',
-    #         'USER': 'na_db_user',
-    #         'PASSWORD': 'na_user',
+    #         'NAME': 'ffw',
```
Comment (nherriot) [inline noisyatom/config/settings.py:44]: I would suggest not putting your personal passwords of your DB into an open project. This will eventually be picked up by the new git_commit software test tools when we switch them on. But if this is just a test DB on your local machine then sure, i have no issues.

File: noisyatom/blogs/forms.py [hunk 7]
```diff
@@ -1,13 +1,14 @@
 from django import forms
-
-from pagedown.widgets import PagedownWidget
-
+#from django.forms import extras
+#from pagedown.widgets import PagedownWidget
+from pagedown.widgets import AdminPagedownWidget
```
Comment (nherriot) [inline noisyatom/blogs/forms.py:4]: Why did you remove those libraries?

File: noisyatom/requirements.txt [hunk 8]
```diff
@@ -1,5 +1,5 @@
 appdirs==1.4.0
-Django==1.10.5
+Django==2
```
Comment (nherriot) [inline noisyatom/requirements.txt:2]: Shall we set this to 2.2 ? Your call. As long as it runs up the web server I don't care. Just let everyone know your decision. Thank you.

**Review by nherriot — CHANGES_REQUESTED** [review]
Nice work.
Just some questions on some parts, then fix merge conflicts and I think this can be pulled into master.

File: noisyatom/catalog/models.py (hunk 5 above)
Comment (tine3700) [reply noisyatom/catalog/models.py:115]: Thanks! I did remove the old code. It won't work anymore with Django 2.2.

File: noisyatom/catalog/models.py (hunk 3 above)
Comment (tine3700) [reply noisyatom/catalog/models.py:32]: Thanks! I did remove the old code. It won't work anymore with Django 2.2.

File: noisyatom/blogs/forms.py (hunk 7 above)
Comment (tine3700) [reply noisyatom/blogs/forms.py:4]: This is an issue which pops up later. It should not be in this PR. I will remove the changes.

File: noisyatom/requirements.txt (hunk 8 above)
Comment (tine3700) [reply noisyatom/requirements.txt:2]: Good call! I will change to Django==2.2

File: noisyatom/config/settings.py [hunk 9]
```diff
@@ -38,25 +38,25 @@
...
+             'USER': 'ffw_user',
+             'PASSWORD': '1234',
+             'HOST': '127.0.0.1',
+             'PORT': '5432',
+         }
+     }
 
     # Making our dev machines use sqlite for now. It makes running up a dev instance from scratch quicker for demo
     # purposes.
-    DATABASES = {
```
Comment (nherriot) [inline noisyatom/config/settings.py:54]: Can you add in the sql-lite back for developers :-) 
It's fine for you to leave your code in here as it does nothing.

Comment (tine3700) [reply noisyatom/config/settings.py:54]: No problem. Changed back to use sql-lite db.

**Review by nh-accedian — APPROVED** [review]
